rs/ethereum/ledger-suite-orchestrator/src/scheduler/mod.rs

```diff
@@ -330,18 +331,18 @@ impl UpgradeLedgerSuiteBuilder {
 #[derive(Clone, Eq, PartialEq, Ord, PartialOrd, Debug, Deserialize, Serialize)]
 pub enum UpgradeLedgerSuiteSubtask {
     UpgradeIndex {
-        contract: Erc20Token,
+        token_id: TokenId,
```

Comment:
Note that all subtasks in `UpgradeLedgerSuiteSubtask` are stored in stable memory, so changing from `Erc20Token` to `TokenId` would be problematic if there was an instance of such a task when the upgrade would occur. However, this is not the case, because those tasks are only there, when a token is added, and consumed once the new ledger suite was spawned off. This is for example visible by the metrics `ledger_suite_orchestrator_tasks` (the 2 tasks there are periodic and correspond to cycles top-up and archive discovery)